src/classes/HarmonizedClient.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) One Beyond. All rights reserved. Licensed under the MIT license.
 
-import { type HarmonizedOutput, type ClientParams, type ClientInstance } from './@types';
-import { AbstractClient } from './index.abstract';
+import { ClientInstance, ClientParams, HarmonizedOutput } from '../@types';
```

Comment:
Are we removing `type` for some reason?

Reply:
The types are imported in the line 3 
`import { ClientInstance, ClientParams, HarmonizedOutput } from '../@types';`

Reply:
Yeah, my question is that before we were importing ONLY the type, as we don't need anything else:

`import { type HarmonizedOutput, type ClientParams, type ClientInstance } from './@types';`

And now you're importing them without the types:

`import { ClientInstance, ClientParams, HarmonizedOutput } from '../@types';
`

More info [here](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export).

Reply:
Honestly, I think the previous import didn't make sense, because you are importing what you are exporting. I mean, It doesn't matter if its a type, an interface or simply a function. You just import it and that's it